### DISCUSSION TITLE
关于poe上的GPT4-32k

### DISCUSSION THREAD

**Discussion by q2408808** [viopsa233/AI-testers-Forum#87] [交流]
昨天用起来是真的爽，能明显感觉到要远强于claude2。但是今天不知道是我的错觉，还是后台改数据了，可以明显感觉到质量的下滑。当然poe上的克2也有点打折扣的感觉。总之我用平时用的prompt来测试，GPT4-32k的回答质量不如官方网页版的克2，更别说早期的sydney了。

**Comment by Rudyjames** [comment]
32k主要是来干嘛的啊

**Reply by q2408808** [reply]
有些详细点的prompt就有几千的token了。不是32K这种等级的上下文，根本不够用。
